packages/extension/src/background/index.ts

```diff
@@ -131,6 +131,25 @@ async function handleMessages(message, sender: Runtime.MessageSender) {
       },
     });
   }
+
+  if (message.type === 'REQUEST_UPDATE') {
```

Comment:
Sure thing, did not want to make a change because its better to have a PR changes focused on the features, I can make a separate PR with enum adjustment and merge it before this one?

Reply:
Definitely, the right thing to do. So long as it will be remembered, you can even create a ticket so it won't be forgotten.

Reply:
Added in #1613